rails/benchmarks/bm_activemodel_views.rb

```diff
@@ -2,8 +2,6 @@
 
 require_relative 'support/benchmark_rails.rb'
 
-require 'ffaker'
-
 require 'rails'
 require 'action_controller/railtie'
 require 'active_model'
```

Comment:
Regarding this benchmark, I think we can still move some of the setup out of the block that we pass to `Benchmark.rails`. 

For instance `current = File.expand_path File.dirname(__FILE__)` is benchmarking Ruby rather than the action that we're concerned with. Thoughts? :smile_cat: